.vscode/templates.code-snippets

```diff
@@ -0,0 +1,204 @@
+{
+	// Place your DroidControlShip workspace snippets here. Each snippet is defined under a snippet name and has a scope, prefix, body and 
+	// description. Add comma separated ids of the languages where the snippet is applicable in the scope field. If scope 
+	// is left empty or omitted, the snippet gets applied to all languages. The prefix is what is 
+	// used to trigger the snippet and the body will be expanded and inserted. Possible variables are: 
+	// $1, $2 for tab stops, $0 for the final cursor position, and ${1:label}, ${2:another} for placeholders. 
+	// Placeholders with the same ids are connected.
+	// Example:
+	// "Print to console": {
+	// 	"scope": "javascript,typescript",
+	// 	"prefix": "log",
+	// 	"body": [
+	// 		"console.log('$1');",
+	// 		"$2"
+	// 	],
+	// 	"description": "Log output to console"
+	// }
+	"Simple Comment": {
+		"prefix": "comment",
+		"scope": "c, cpp",
+		"body": [
+			"/* ${1:Comment} */"
+		],
+		"description": "Simple C-styled comment."
+	},
+	"Doxygen In-Line Comment": {
+		"prefix": "doxy_inline",
+		"scope": "c, cpp",
+		"body": [
+			"/**< ${1:Comment} */",
+		],
+		"description": "Simple in-line Doxygen comment."
+	},
+	"Doxygen Comment": {
+		"prefix": "doxy_comment",
+		"scope": "c, cpp",
+		"body": [
+			"/** ${1:Comment} */",
+			"$0"
+		],
+		"description": "Simple Doxygen comment."
+	},
+	"Doxygen Block Comment/Description": {
+		"prefix": "doxy_block",
+		"scope": "c, cpp",
+		"body": [
+			"/**",
+			" * ${1:description}",
+			" * ",
+			" * ${2:parameters}",
+			" * ${3:returns}",
+			" */",
+			"$0"
+		],
+		"description": "Doxygen comment/description block."
+	},
+	"Template Header File": {
+		"prefix": "template_header",
+		"scope": "c, cpp",
+		"body": [
+			"/* MIT License",
+			" *",
+			" * Copyright (c) 2023 Andreas Merkle <web@blue-andi.de>",

```

Comment:
Can the year be automatically set to the current year.

Reply:
Not automatically, no. We could make it variable, and each time we have to type the year. Or we change the template each year. Or make an script of some kind to run on the CI

Reply:
Forget it, just found out how